lib/expand/bootstrap.go

```diff
@@ -200,3 +205,17 @@ func getPeerAddrAndToken(ctx operationContext, role string) (peerAddr, token str
 	}
 	return peerAddr, instructions.Token, nil
 }
+
+func patchSELinuxConfig(ctx operationContext) error {
```

Comment:
Could you add a comment about what this function does - "patch selinux config" doesn't tell much.